README.md

```diff
@@ -0,0 +1,50 @@
+# Calculate Journey Variable Payments
+
+## Prerequisites
+
+- [Docker and Docker compose](https://docs.docker.com/get-docker/)
+- [Intellij IDEA](https://www.jetbrains.com/idea/)
+
+## Running locally
+
+You can run the dependencies for the projects with Docker compose:
+
+```bash
+docker-compose up
+```
+
+The command will launch:
+
+- [Localstack](https://github.com/localstack/localstack) (which is used to mock S3)
+- Postgres
+
+Next, you should load the mock data into the Localstack S3.
+
+First, retrieve the ID of the current Localstack container.
+
+```bash
+$ docker ps
+CONTAINER ID        IMAGE                          NAMES
+a51bc3adcd34        localstack/localstack:0.11.5   localstack
+bdae7e94639a        postgres:latest                jpc-database
+```
+
+Attach to the Localstack container with:
+
+```bash
+docker exec -ti <a51bc3adcd34> bash
+```
+
+Load the data with:
+
+```bash
+/docker-entrypoint-initaws.d/setup.sh
```

Comment:
`docker-compose up` automatically does this for you so no need to connect and load manually.  Well it should do this.  Does it not for you?

Reply:
I couldn't see where the `setup.sh` file is called. Is it executed by the Localstack Docker container?

Reply:
it happens here automagically here in the compose file -   `- $PWD/localstack:/docker-entrypoint-initaws.d`